src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -101,6 +100,12 @@ public synchronized void addJournalEntry(JournalEntry journalEntry) throws Excep
         indicateJournalChanged();
     }
 
+    @Override
+    public synchronized void deletePerson(ReadOnlyPerson target) throws PersonNotFoundException {
+        addressBook.removePerson(target);
+        indicateAddressBookChanged();
```

Comment:
@traceurgan need to refactor this? to indicatePartnerChanged() or something then rename Person into Partner?

Reply:
Done. Slightly altered the implementation of add, edit, delete. Now they all go through update person since all the the commands just need to update person from null to person (add), person to person(edit) or person to null (delete). can change back if need be also. Indicate person changed is also done once through successful update person since all the them now call update person